apps/trading/pages/client-router.tsx

```diff
@@ -199,6 +204,32 @@ export const useRouterConfig = (): RouteObject[] => {
         },
       ],
     },
+    {
```

Comment:
Can we:
- Add feature flag for all the AMM pagse
- Wrap the root element with a component to load rest markets and assets with suspense so we don't fetch everything twice for the whole app